src/NuGet.Clients/PackageManagement.VisualStudio/ProjectSystems/LegacyCSProjPackageReferenceProject.cs

```diff
@@ -278,26 +281,35 @@ private PackageSpec GetPackageSpec()
 
             // In legacy CSProj, we only have one target framework per project
             var tfis = new TargetFrameworkInformation[] { projectTfi };
-            return new PackageSpec(tfis)
+
+            return new PackageSpec()
             {
                 Name = _projectName ?? _projectUniqueName,
+                Version = new NuGetVersion(1, 0, 0, 0, (string)null, string.Empty),
+                Authors = new string[] { },
+                Owners = new string[] { },
+                Tags = new string[] { },
+                TargetFrameworks = tfis,
+                ContentFiles = new string[] { },
+                Dependencies = packageReferences,
+                IsDefaultVersion = true,
                 FilePath = _projectFullPath,
                 RuntimeGraph = runtimeGraph,
                 RestoreMetadata = new ProjectRestoreMetadata
                 {
-                    OutputType = RestoreOutputType.NETCore,
+                    OutputType = RestoreOutputType.UAP,
```

Comment:
This should stay as NETCore, only the UWPProjectJson should use this type from how I understand the goal.